gradle/scala.gradle

```diff
@@ -91,6 +99,11 @@ export H2O_EXTENDED_JAR=`./gradlew -q extendJar -PdownloadH2O`
     }
 }
 
+// Override ScalaDoc classpath to look only for classes in the scala output dir preventing several warnings for
+// missing elements since we don't use java source dirs
+scaladoc {
```

Comment:
note - it doesn't mean we don't generate javadoc for java classes, the scaladoc will just doesn't look in the empty java output directory since we are using scala output directory